include/camp/defines.in.hpp

```diff
@@ -106,6 +105,10 @@ namespace camp
 #define CAMP_SUPPRESS_HD_WARN
 #endif
 
+#if defined( CAMP_ENABLE_OPENMP ) && defined(_OPENMP)
+#define CAMP_HAVE_OPENMP
+#endif
+
 #if defined(ENABLE_TARGET_OPENMP)
```

Comment:
I think this needs to use the new internal cmakedefine'd flag. At least this works for me when trying out https://github.com/LLNL/RAJA/issues/1111 and https://github.com/LLNL/RAJA/issues/1145 

```suggestion
#if defined(CAMP_ENABLE_TARGET_OPENMP)
```

Reply:
Thanks for that, you're right.